apps/agentstack-ui/.env.example

```diff
@@ -22,7 +22,7 @@ NEXT_PUBLIC_APP_FAVICON_SVG=
 
 # Default false
 OIDC_ENABLED=
-OIDC_PROVIDERS='[{"name": "w3id","id": "w3id","type": "oidc","client_id": "<your_client_id>","client_secret": "<your_client_secret>","issuer": "your_issuer"}]'
```

Comment:
This exists to keep backwards compatibility. `type` is no longer needed, newly added custom is discriminated union for scehma validation.